modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -16,6 +16,36 @@ New features are added to the language continuously, and occasionally, some feat
 This section lists all of the features that have been removed, deprecated, added, or extended in different Cypher versions.
 Replacement syntax for deprecated and removed features are also indicated.
 
+[[cypher-deprecations-additions-removals-5.26]]
+== Neo4j 5.26
+
+=== Deprecated features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+|===
+
+=== New features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role="noheader"]
+----
+GRANT READ {*} ON GRAPH * FOR (n) WHERE n.createdAt > date() TO regularUsers
+----
+| link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/property-based-access-control/[Property-based access control] does now support temporal and point values.
```

Comment:
might be nice to also mention here that they are not dynamic but evaluated at grant/deny time?

Reply:
Hmm, maybe I should have `date('2024-10-25')` as an example instead? Will probably cause less confusion

Reply:
Feels like this page should not contain that much detail, if you want to read more there is a link to the proper documentation of the feature.